code/aepcomposeui/src/main/java/com/adobe/marketing/mobile/aepcomposeui/observers/AepUiEventObserver.kt

```diff
@@ -0,0 +1,7 @@
+package com.adobe.marketing.mobile.aepcomposeui.observers
+
+import com.adobe.marketing.mobile.aepcomposeui.interactions.UIEvent
+
+interface AepUiEventObserver {
```

Comment:
Same comment here. Let's add a description for the interface and what `onEvent` is for